server/app/views/components/ProgramCardFactory.java

```diff
@@ -72,7 +72,12 @@ public DivTag renderCard(Request request, ProgramCardData cardData) {
                                 "font-bold",
                                 "text-xl"))
                     .with(
-                        p(programDescriptionText)
+                        div()
+                            .with(
+                                TextFormatter.formatText(
```

Comment:
This formats the program description for admin on the page that lists all their programs 
<img width="923" alt="image" src="https://github.com/civiform/civiform/assets/24319951/8304b7dc-ca36-409a-9134-adb722421e70">